backend/requirements/base.txt

```diff
@@ -1,3 +1,4 @@
-Django==1.11
```

Comment:
The version of wagtail I used when I started the integration wasn't compatible with django 1.11 yet